website/markdown/posts/2020-03-25-version-1.5.0/post.mdx

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: Tuist 1.5.0 - Scaffold command, performance, mint
+categories: [tuist, release, swift, project generation, xcode, apple, "1.4.0"]
+excerpt: Tuist's new version 1.5.0 brings scaffold command, performance improvements and mint support
+author: fortmarek
+---
+
+Tuist is being worked on even from home 🏡 and I am stoked to present to you what's new.
+It's not even so long ago that the version 1.4.0 has been released, but there's already enough new improvements to justify a new one!
+That includes a brand new command `scaffold`, or [mint](https://github.com/yonaskolb/mint) suppport, besides new features we are working hard on making generation faster and more performant 🏎
+Let's see in detail what this release brings.
+
+## Scaffold command
+
+A lot of teams or even individuals have some common structure for their features or components,
+but creating a new one has meant using a custom solution or copy&paste,
+which is tedious and takes away time that developers can focus on coding.
+
+To fix this common problem we are introducing a new `scaffold` command.
+Now you can run `tuist scaffold framework --name MyFramework`
+and tuist automatically generates all files and folders that you define in a new manifest called `Template.swift`.
+To read more about how it works you can do so on [commands](/docs/usage/commands/) documentation page.
+
+We are excited too see what *you* will be able to do with it!
```

Comment:
What about including some examples of how teams could use this feature?

- Bootstrap new features (code) that follow the project architecture: `e.g. tuist scaffold viper --name Home`
- Create new feature frameworks (i.e. a Project.swift and some source files)
- ....

That'd give readers an idea of how the feature can be used.